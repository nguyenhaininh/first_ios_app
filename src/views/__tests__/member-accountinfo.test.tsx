/*---------------------------------------------------------------------------------------------
 *  Copyright (c) Lunascape Corporation. All rights reserved.
 *--------------------------------------------------------------------------------------------*/
import * as React from 'react';
import MemberAccountInfo from '../member-accountinfo';

import * as renderer from 'react-test-renderer';

it('renders correctly', () => {
  const fn = jest.fn();
  const tree = renderer.create(
    <MemberAccountInfo />
  ).toJSON();
  expect(tree).toMatchSnapshot();
});
