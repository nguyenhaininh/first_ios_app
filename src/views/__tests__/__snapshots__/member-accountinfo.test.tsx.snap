// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View
  style={
    Object {
      "backgroundColor": "#fff",
      "flex": 1,
    }
  }
>
  <Text
    accessible={true}
    allowFontScaling={true}
    ellipsizeMode="tail"
  >
    T.B.D
  </Text>
  <Text
    accessible={true}
    allowFontScaling={true}
    ellipsizeMode="tail"
  />
</View>
`;
